test/simulator/main/org/apache/cassandra/simulator/ClusterSimulation.java

```diff
@@ -683,9 +688,10 @@ public ClusterSimulation(RandomSource random, long seed, int uniqueNum,
                              .set("memtable_allocation_type", builder.memoryListener != null ? "unslabbed_heap_buffers_logged" : "heap_buffers")
                              .set("file_cache_size", "16MiB")
                              .set("use_deterministic_table_id", true)
-                             .set("disk_access_mode", "standard")
+//                             .set("disk_access_mode", "standard")
```

Comment:
wondering if it would be good to have these configs based off the `RandomSource`?  that way we test all code paths rather than specific ones?